cmd/validate/image_integration_test.go

```diff
@@ -47,45 +44,23 @@ import (
 func TestEvaluatorLifecycle(t *testing.T) {
 	noEvaluators := 100
 
+	// Clear the download cache to ensure a clean state for this test
+	source.ClearDownloadCache()
+
 	ctx := utils.WithFS(context.Background(), afero.NewMemMapFs())
 	client := fake.FakeClient{}
 	commonMockClient(&client)
 	ctx = oci.WithClient(ctx, &client)
 	mdl := MockDownloader{}
-	downloaderCall := mdl.On("Download", mock.Anything, mock.Anything, false).Return(&ociMetadata.OCIMetadata{Digest: "sha256:da54bca5477bf4e3449bc37de1822888fa0fbb8d89c640218cb31b987374d357"}, nil).Times(noEvaluators)
+	// The downloader should be called twice per policy source: once during PreProcessPolicy and once during evaluator evaluation
```

Comment:
Are we talking about a go-gather downloader here? Why does it get downloaded twice now?

Reply:
I had a lot of problems with this test and part of the solution at one point was to clear the policy cache which made it so the policy was downloaded twice. It's downloaded once in `PreProcessPolicy` and again in the `Evaluate` function.
I see now that the cache doesn't need to be cleared, so I'll fix that.